content/800-data-platform/850-billing/06-data-proxy-usage.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Data Proxy usage notifications'
+metaTitle: 'Data Proxy usage notifications'
+metaDescription: 'Prisma Data Platform continuously monitors the Data Proxy usage of your projects and sends automated notifications to the email address of your GitHub account when your project reaches usage milestones.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Prisma Data Platform continuously monitors the Data Proxy usage of your projects and sends automated notifications to the email address of your GitHub account when your project reaches usage milestones.
+
+</TopBlock>
+
+## Milestones that trigger notifications
+
+You receive automated email notifications based on your project plan and on the amount of Data Proxy usage you reach during any given month.
```

Comment:
Update beginning of sentence to "You will receive...". Thanks!

Reply:
Thanks for the rewording. That sounds better. I'm not an English language grammar expert -- but I think changing the subject from "You" to "Prisma Data Platform" makes it sound more correct when using present simple tense. I appreciate that we have a style guide and we adhere to it! Thanks!